### ISSUE TITLE
Has anyone consider adding an Internet of Things category?

### ISSUE THREAD

**Issue by drborges** [trecenti/javascript-toolkit#16]
IoT is becoming more and more trendy as applications become more pervasive in our lives. In case you have interest of creating this category, here are some cool javascript libraries that would fit in IMHO:

- http://cylonjs.com
- http://devicejs.org
- https://github.com/ecto/duino

**Comment by trecenti** [comment]
yes! we're considering expanding it to IoT and other platforms / technologies as well, thanks!
